packages/ag-charts-community/src/chart/layers.ts

```diff
@@ -7,7 +7,6 @@ export enum Layers {
     AXIS_ZINDEX = 20,
     SERIES_CROSSLINE_RANGE_ZINDEX = 30,
     SERIES_LAYER_ZINDEX = 500,
-    SERIES_ERRORBAR_ZINDEX = 600,
```

Comment:
Note: this is no longer necessary now that the error bar are in their own group.